assets/translations.js

```diff
@@ -141,7 +141,7 @@ export default {
         arabic: "شارك خدمة INFOSforyou مع أصدقائك",
         persian: "بسيار خوب ! به زودى اينفوز فور يو جديدترین اخبار را برای شما خواهد فرستاد.",
         english: "Share INFOSforyou with your friends.",
-        german: "Teile den Service INFOSforyou mit Deinen Freunden",
+        german: "Teile  INFOSforyou mit deinen Freunden",
```

Comment:
Why is there an extra space here?

Reply:
It aligned nicely in the code with the english translation.